docs/next/guides/rows/row-sorting.md

```diff
@@ -289,20 +289,42 @@ beforeColumnSort(currentSortConfig, destinationSortConfigs) {
 The plugin provides two hooks:
 
 * [beforeColumnSort](@/api/hooks.md#beforecolumnsort) runs before the sort
-  * The sort configuration obtained by the `getSortConfig` method within the callback will match the sort configuration preserved before the hook call
-  * The callback for `beforeColumnSort` will return `false` and stop the table from being sorted, which results in the `afterColumnSort` hook not being called
-* [afterColumnSort](@/api/hooks.md#aftercolumnsort) always runs after sorting unless the callback for `beforeColumnSort` hook returns `false`
+  * The sort configuration obtained by the [`getSortConfig`](@/api/columnSorting.md#getsortconfig) method within the callback will match the sort configuration preserved before the hook call
+  * The callback for [`beforeColumnSort`](@/api/hooks.md#beforecolumnsort) will return `false` and stop the table from being sorted, which results in the [`afterColumnSort`](@/api/hooks.md#aftercolumnsort) hook not being called
+* [afterColumnSort](@/api/hooks.md#aftercolumnsort) always runs after sorting unless the callback for [`beforeColumnSort`](@/api/hooks.md#beforecolumnsort) hook returns `false`
 
 ::: tip
-Hooks are also run when you use the `clearSort` method or provide a configuration that won't be processed, causing validation to fail.
+Hooks are also run when you use the [`clearSort`](@/api/columnSorting.md#clearsort) method or provide a configuration that won't be processed, causing validation to fail.
 :::
 
 ## Plugin API
 
 List of methods exposed by the plugin:
 
-* [`clearSort` &#8594;](@/api/columnSorting.md#clearsort)
-* [`getSortConfig` &#8594;](@/api/columnSorting.md#getsortconfig)
-* [`isSorted` &#8594;](@/api/columnSorting.md#issorted)
-* [`setSortConfig` &#8594;](@/api/columnSorting.md#setsortconfig)
-* [`sort` &#8594;](@/api/columnSorting.md#sort)
+* [`clearSort`](@/api/columnSorting.md#clearsort)
+* [`getSortConfig`](@/api/columnSorting.md#getsortconfig)
+* [`isSorted`](@/api/columnSorting.md#issorted)
+* [`setSortConfig`](@/api/columnSorting.md#setsortconfig)
+* [`sort`](@/api/columnSorting.md#sort)
+
+## Related articles
+
+#### Related guides
+
+- [Column sorting](@/guides/columns/column-sorting.md)
+
+#### Related blog articles
+
+- [Feature spotlight: Multi-column sorting](https://handsontable.com/blog/feature-spotlight-multi-column-sorting)
```

Comment:
Check with Ola

Reply:
Ola says that for now it's OK (but they'll be double-checking blog posts soon).